lib/google_scraping/dashboard/queries/keyword_query.ex

```diff
@@ -6,5 +6,8 @@ defmodule GoogleScraping.Dashboard.Queries.KeywordQuery do
   @doc """
   Returns the keywords list for the given user_id
   """
-  def user_keywords_query(user_id), do: where(Keyword, [k], k.user_id == ^user_id)
+  def user_keywords(user_id, search_phrase \\ nil) do
+    wildcard_search = "%#{search_phrase}%"
+    Keyword |> where([k], k.user_id == ^user_id) |> where([k], ilike(k.name, ^wildcard_search))
```

Comment:
Minor comment but piping vertically makes it easier to read

```suggestion
    wildcard_search = "%#{search_phrase}%"

    Keyword 
    |> where([k], k.user_id == ^user_id) 
    |> where([k], ilike(k.name, ^wildcard_search))
```